flow-typed/react-intl_vx.x.x.js

```diff
@@ -61,22 +96,25 @@ declare module 'react-intl' {
     pattern: string;
     location?: Location_2;
   }
-  declare var DEFAULT_INTL_CONFIG: Pick<
+  declare var DEFAULT_INTL_CONFIG: SubsetProperties<
     IntlConfig,
-    | 'formats'
-    | 'messages'
-    | 'timeZone'
-    | 'textComponent'
-    | 'defaultLocale'
-    | 'defaultFormats'
-    | 'onError', >;
+    {|
+      formats: mixed,
+      messages: mixed,
+      timeZone: mixed,
```

Comment:
This isn't quite right -- to match the intended behavior it needs to be `timeZone?: mixed`, unfortunately:
```
 * A limitation: this implementation can't tell which properties are
 * *optional* in T, i.e. spelled with `?:`, as in `foo?: number`.  Instead,
 * properties in the result will be optional or required based on how they
 * are in U.  To avoid confusion, be sure to make the same properties
 * optional in U as are optional in T.
```
I.e., `IntlConfig` has `timeZone` optional, so this needs to have `timeZone` optional as well in order to make it optional in the result.

(Similarly some other properties.)

Reply:
Thanks for catching this! I think I've got them all in the new revision.